index.html

```diff
@@ -0,0 +1,89 @@
+    </section>
+    <nav>
+      <div class="logo"><a href="index.html"><strong>E</strong>asy <strong>S</strong>ience</a></div>
+      <div class="hamburger-menu"><img id="menu-button" src="assets/hamburger-menu.png" alt="menu"></div>
+      <ul id="menu">
+        <li class="closeButtonContainer"><span id="closeButton">X</span></li>
+        <li><a class="Personal-information" href="courses.html">Courses</a></li>
+        <li><a class="Personal-information" href="#about">About</a></li>
+        <li><a class="Personal-information" href="#contact">Contact</a></li>
+        <li><a class="Personal-information button sign-up" href="#">Sign up</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="home-page">
+        <div class="home-text">
+          <h1>We teach science the right way</h1>
+          <p>We have the mission to provide a world-class education for anyone, anywhere.</p>
+          <a class="button" href="courses.html">Start here</a>
+        </div>
+    </section>
+    <section id="about" class="about">
+      <h2>Find a comprehensive explanation for the subject you've been struggling with</h2>
+      <hr>
+      <div>
+        <p>Start with:</p>
+        <ul>
+          <li class="button"><a href="#">Math   <i class="fas fa-square-root-alt"></i></a></li>
+          <li class="button"><a href="#">Physics   <i class="fas fa-atom"></i></a></li>
+          <li class="button"><a href="#">Chemistry   <i class="fas fa-flask"></i></a></li>
+        </ul>
+      </div>
+    </section>
+    <section id="courses" class="courses-container">
+      <h1>Take a look at our most popular courses</h1>
+      <hr>
+      <section id="courses-wrap" class="courses">
+        <div class="button-container">
+          <a class="button show-more" href="#">Show more   <i class="fas fa-arrow-down"></i></a>
+          <a class="button" href="courses.html">Find more courses   <i class="fas fa-book-reader"></i></a>
+        </div>
+      </section>
+    </section>
+    <section id="contact" class="contact-container">
+      <div class="contact-text">
+        <h2>Need a tutor?</h2>
+        <p>Contact us for a personalized class</p>
+        <hr>
+      </div>
+      <div class="contact-form">
+        <form id="form" class="form" action="#" method="post">
+          <input type="text" id="name" name="user-name" placeholder="Full Name" required maxlength="30">
+          <input type="email" id="email" name="user-email" placeholder="Email Address" required>
+          <button id="button" class="button" type="submit">Get in Touch</button>
+        </form>
+        <p>or</p>
+        <a class="button" href="#">Sign up</a>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <section class="author">
+      <p>copyright© 2021 easyscience.com</p>
+      <div class="logo"><a href="index.html"><strong>E</strong>asy <strong>S</strong>ience</a></div>
+      <p>All right reserved. Designed by Carlos Herver</p>
+    </section>
+  </footer>
```

Comment:
### Home Footer

- [ ] [REQUIRED] As per project requirements the footer should contain a logo aligned to the left and text aligned to the right and the design should match the one below.

- [ ] [REQUIRED] On the Mobile - It has a white background.

![image](https://user-images.githubusercontent.com/4247740/147304164-fc1bc88c-f622-4261-87a9-00eda316d8a4.png)

- [ ] [REQUIRED] On the Desktop - It has a dark background.

![image](https://user-images.githubusercontent.com/4247740/147304188-00de37eb-0acb-4df3-939c-440c7a02969c.png)